src/install/ibm-mq/linux/requirements-linux.mdx

```diff
@@ -0,0 +1,11 @@
+---
+componentType: default
+headingText: Check the compatibility and requirements
+---
+
+* [Install the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent-new-relic), minimum version 1.24.0.
```

Comment:
I'd pull this out into it's own step: https://docs.newrelic.com/install/mongodb/#install-the-infrastructure-agent-3